JavaScript/JavaScriptSDK/Telemetry/Common/DataSanitizer.ts

```diff
@@ -156,6 +154,27 @@ module Microsoft.ApplicationInsights.Telemetry.Common {
             return measurements;
         }
 
+        public static sanitizeId(id: string): string {
+                return id ? DataSanitizer.sanitizeInput(id, DataSanitizer.MAX_ID_LENGTH, _InternalMessageId.IdTooLong).toString() : id;
+        }
+
+        public static sanitizeInput(input: any, maxLength: number, _msgId: _InternalMessageId) {
```

Comment:
`sanitizeInput()` is indirectly tested with `sanitizeUrl()` and `sanitizeId()` - would you like to explicitly test it as well?
It's in public API and one would not know that it's untested if one is to take dependency on this method later.

Reply:
Added unit tests.